lib/series_external_relationship.rb

```diff
@@ -123,7 +123,7 @@ def aremos_comp_display_array
         end
       end
       results
-    rescue Exception => e
+    rescue Exception,StandardError => e
```

Comment:
Since StandardError inherits from Exception, this seems redundant.

Reply:
Right... and glad to say that I noticed and fixed it myself before seeing your comment :=)